cli/cmd/testdata/inject_gettest_deployment.input

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: get-test
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: get-test-deploy-injected-1
+  namespace: get-test
+spec:
+  replicas: 1
+  selector:
+      matchLabels:
+        app: get-test
```

Comment:
apologies did not noticed this on first review, can you shrink this file down to something simpler that yields the same error, for example:
```yaml
---
kind: Deployment
spec:
  template:
      metadata:
        labels:
          app: get-test
    spec:
      containers: []
```

Reply:
Yep, can definitely do that. Thanks for the suggestion

Reply:
> can you shrink this file down to something simpler that yields the same error

We need a test case where there is at least one valid object before the invalid object.